source/Irrlicht/CMakeLists.txt

```diff
@@ -139,6 +142,10 @@ if(NOT (BSD OR SOLARIS OR EMSCRIPTEN))
 	add_definitions(-D_IRR_COMPILE_WITH_JOYSTICK_EVENTS_)
 endif()
 
+# EGL
+
+set(ENABLE_EGL OFF)
```

Comment:
there are `set(ENABLE_EGL)` calls above this line, the value will be reset

Reply:
ah yeah right it's too upper my bad

Reply:
fixed